src/nodes/materials/MeshLambertNodeMaterial.js

```diff
@@ -21,9 +21,11 @@ class MeshLambertNodeMaterial extends NodeMaterial {
 
 	}
 
-	setupLightingModel( /*builder*/ ) {
+	setupLightingModel( builder ) {
 
-		return new PhongLightingModel( false ); // ( specular ) -> force lambert
+		const envNode = this.getEnvNode( builder );
+
+		return new PhongLightingModel( envNode, false, ); // ( specular ) -> force lambert
```

Comment:
`( envNode, false, )` 👀

Reply:
Should be fixed now! 😇

Reply:
Are you sure? I still see `false, )` 👀

Reply:
For `MeshLambertNodeMaterial` the `specular` parameter must be `false`.

I assumed your comment was related to the missing `combine` parameter which I have added in a separate commit.